h_transport_materials/property_database/molybdenum.py

```diff
@@ -49,12 +49,25 @@
     source="frauenfelder_permeation_1968",
 )
 
+guthrie_p_0 = 3.8e-3 * u.ccSTP * u.cm**-1 * u.s**-1 * u.atm**-0.5
+guthrie_permeability = Permeability(
+    pre_exp=guthrie_p_0 / ((htm.Rg * 300 * u.K),
+    act_energy=17.4 * u.kcal * u.mol**-1,
+    isotope="D",
+    range=(
+        u.Quantity(270, u.degC),
+        u.Quantity(640, u.degC),
+    ),
+    source="guthrie_permeation_1974",
```

Comment:
```suggestion
    source="guthrie_permeation_1974",
    note="Figure 7",
```

It is always good practice to note where to find the information in the paper